python/tests/test_cats.py

```diff
@@ -17,3 +17,23 @@ def test_cats():
     assert action >= 10
     assert action <= 20
     vw.finish()
+
+def test_cats_pdf():
+    min_value = 10
+    max_value = 20
```

Comment:
Can you add a comment explaining how you chose these values?

Reply:
I used the same setup used by the existing `test_cats`  function in the file but changed `--cats` to `--cats_pdf`. But I'm not sure how those values were picked.

Reply:
It doesn't really matter what these values are, they were picked randomly in the existing cats test. Cats needs min and max to determine the action range and then we can use them to test the prediction is not out of the supplied range.